scripts/check_ledger_health.js

```diff
@@ -0,0 +1,197 @@
+/*
+ * This script runs through a few checks and lets us know if something is off
+ * Note, best to run this script on a prod_snapshot locally, until we create a read-only follower
```

Comment:
even so, we should run those scripts on a snapshot to not slow down production db.

Reply:
Doesn't matter anymore - script takes less than half a second to run. I'm going to set it up as an automatic cron job after system is stable, so it can email us when it finds discrepancies.